_release/5.0.0.md

```diff
@@ -0,0 +1,762 @@
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Apache Arrow 5.0.0 (29 July 2021)
+
+This is a major release covering more than 3 months of development.
+
+## Download
+
+* [**Source Artifacts**][1]
+* **Binary Artifacts**
+  * [For CentOS][2]
+  * [For Debian][3]
+  * [For Python][4]
+  * [For Ubuntu][5]
+* [Git tag][6]
+
+## Contributors
+
+This release includes 684 commits from 99 distinct contributors in 2 Arrow repositories.
+
+```console
+77  David Li
+43  Krisztián Szűcs
+42  Antoine Pitrou
+35  Dominik Moritz
+33  Nic Crane
+27  Weston Pace
+27  Sutou Kouhei
+27  Andrew Lamb
+23  Jonathan Keane
+21  Joris Van den Bossche
+17  Jorge C. Leitao
+17  Ian Cook
+16  João Pedro
+15  Jiayu Liu
+14  Yibo Cai
+14  Alessandro Molina
+13  Jorge Leitao
+12  Ritchie Vink
+12  Benjamin Kietzman
+10  niranda perera
+10  Wakahisa
+10  Matthew Topol
+9  Anthony Louis
+8  Raphael Taylor-Davies
+8  Nate Clark
+7  Eduardo Ponce
+6  Daniël Heres
+5  Neal Richardson
+5  Navin
+5  Jörn Horstmann
+5  Andy Grove
+4  crystrix
+4  Uwe L. Korn
+4  Romain Francois
+4  Rok
+4  Marco Neumann
+4  Ádám Lippai
+3  liyafan82
+3  frank400
+3  Ying Zhou
+3  Roee Shlomo
+3  Michael Edwards
+3  Matthijs Brobbel
+3  Alenka Frim
+2  Zachary Blackwood
+2  Wes McKinney
+2  Steven
+2  P42
+2  Michal Nowakiewicz
+2  Kazuaki Ishizaki
+2  Karik Isichei
+2  Jeroen Ooms
+2  Gary Pennington
+2  Diana Clarke
+2  Bryan Cutler
+1  sjgupta2
+1  sgilmore
+1  ptaylor
+1  nullptr
+1  kazuhiko kikuchi
+1  hulunbier
+1  baishen
+1  Zimo Zhang
+1  Yordan Pavlov
+1  Will Jones
+1  Steven Burns
+1  Stephen
+1  Shuai Zhang
+1  Ray Bell
+1  Projjal Chanda
+1  Pachamaltese
+1  Nick Bruno
+1  Micah Kornfield
+1  Menno Siekerman
+1  Max Meldrum
+1  Mauricio Vargas
+1  Marc van Heerden
+1  Manish Gill
+1  Maarten A. Breddels
+1  Lorenz Walthert
+1  Linan Qiu
+1  Laurent Mazare
+1  Kornelijus Survila
+1  Kevin Gurney
+1  Joris Peeters
+1  Jinpeng Zhou
+1  Jiangtao Peng
+1  Hideaki Hayashi
+1  Giordon Stark
+1  Gang Liao
+1  GALI PREM SAGAR
+1  Edd Robinson
+1  Dorian Kind
+1  Dmitry Patsura
+1  Chojan Shang
+1  Boaz
+1  Ben Chambers
+1  Alex Baden
+1  Adam Johnson
+```
+
+## Patch Committers
+
+The following Apache committers merged contributed patches to Arrow repositories.
+
+```console
+150  GitHub
```

Comment:
@thisisnic it looks like all the `arrow-rs` commits are listing the committer as `GitHub`

Reply:
Correctly, since the PRs get merged with the `Squash and Merge` button from the Github UI.

Reply:
```
Author:     Author's name <Author's email>
AuthorDate: Thu Jul 29 04:53:14 2021 +1000
Commit:     GitHub <noreply@github.com>
CommitDate: Wed Jul 28 14:53:14 2021 -0400
```

Reply:
@alamb do you know why `git shortlog` outputs `GitHub` as the committer for most of the `arrow-rs` commits?
```
git shortlog --summary --group=committer 4.0.0..5.0.0
     3  Andy Grove
   138  GitHub
    20  Jorge C. Leitao
```

Reply:
Ugh, apparently when the PR author is also the committer, GitHub uses `GitHub <noreply@github.com>` as the committer. For the Arrow monorepo this doesn't happen, I assume because we use the merge script.

Reply:
Shall we just remove this entry for now?

We can handle this scenario in https://issues.apache.org/jira/browse/ARROW-13483 for the next release.

Reply:
I applied a manual fix in https://github.com/apache/arrow-site/pull/133 to add in the actual Rust committers

Reply:
FWIW I ran the command `git shortlog -sn 4.0.0..5.0.0` and got a list that did not include `GitHub`: https://github.com/apache/arrow-site/pull/128 

Sorry this caused trouble and thank you @ianmcook  for sorting it out.

Reply:
Sorry, no idea why I didn't realise this was wrong and flag it up, thanks for sorting it @ianmcook